wiki/pages/Testing-Packages.md

```diff
@@ -0,0 +1,21 @@
+As mentioned in other pages such as [tests](Package-Definition-Guide#tests),
+you can specify tests to run like this:
+
+	]$ rez-test maya_utils lint unittest
+
+This test would run two tests, "lint" and "unittest", from a Rez package called
+"maya_utils". When that happens, the pre-defined commands for "lint" and
+"unittest" will run. However if you to "rez-env" into those test environments,
+you can do that using the *--interactive* flag.
+
+	]$ rez-test maya_utils --interactive lint
+	]$ rez-test maya_utils --interactive unittest
+
+In the example commands above, a "rez-env" is auto-generated to combine
+"maya_utils" and "lint" / "maya_utils" and "unittest" into two, separate
+environments.
+
+Note that you cannot call *--interactive* on more than one Rez test at a time.
+
+The resulting environment is exactly what Rez sees, just before a test
+"command" gets run.
```

Comment:
```suggestion
This command runs two test targets, "lint" and "unittest", from a Rez package called
"maya_utils". Rez will execute each test target command following the order the targets.

It is also possible to "rez-env" into a test environment by using the `--interactive` flag:

	]$ rez-test maya_utils --interactive lint

The resulting environment is the combination of the requirements defined by the `maya_utils`
package and the requirements of the `lint` target and is exactly the same as the one used
when the `--interactive` flag is not provided.

Note that you cannot call `--interactive` on more than one target.
```

Reply:
I find your suggested edit a bit confusing. It removes the second --interactive command but keeps the original `rez-test maya_utils --interactive lint unittest`. Which is fine, but then there's this line.

`Rez will execute each test target command following the order the targets.`
At first I thought that also applied to `--interactive` but of course it can't, since you can only select one test at a time with that flag.

I think I understand what you're trying to suggest in this edit, I'll give a shot at a re-write in a moment. Could you have a look again once I've pushed it?

Reply:
Not sure why GitHub didn't update the PR but FWIW I pushed it here: https://github.com/ColinKennedy/rez/blob/issues/665-add_rez_test_interactive_option_2/wiki/pages/Testing-Packages.md